app.rb

```diff
@@ -0,0 +1,165 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @people.each do |person|
+      puts "ID: #{person.id}, Name: #{person.name}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    puts 'Is the person a Student(1) or a Teacher(2)? (Input the number)'
+    type = gets.chomp.to_i
+
+    case type
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else
+      puts 'Invalid option. Person creation failed.'
+    end
+  end
+
+  def create_book
+    puts "Enter book's title:"
+    title = gets.chomp
+    puts "Enter book's author:"
+    author = gets.chomp
+
+    book = Book.new(title, author)
+    @books << book
+    puts "Book created with ID: #{book.id}"
```

Comment:
- [ ] When I create a book it gives an error and I can't create it, kindly fix that 🙃 

<img width="1135" alt="Screenshot 2023-09-15 at 14 14 14" src="https://github.com/mariegrace31/school_library/assets/74501523/485416dc-da40-432c-8e50-57341887b8fa">

- [ ] Also as we can't create teacher, student, and book I didn’t check for the rental because to create rentals we need to have a book and person already created, so after fixing the problem kindly make sure that we can create rental correctly 😁 😁